### DISCUSSION TITLE
afetharita.com'a enkazdan örnek alındı/alınmadı özelliği

### DISCUSSION THREAD

**Discussion by dengizik** [acikkaynak/deprem-yardim-projesi#84] [Ideas]
Öğrendiğim kadarıyla barolar enkazlar kaldırılmadan örnek alınması ve sonrasında hukuki sürecin başlatılması için hukukçulardan oluşan bir gönüllü ağı kurmaya başlamış. 

afetharita.com'a yıkılan binalardan örnek alındı mı, alınmadı mı; hangi binalardan daha örnek almaya ihtiyaç var vb konularında bir özellik eklenebilir.

Toplanan verilerin tam olarak belli bir enkazdan alındı; şu saatte, şu günde, şu kişi tarafından alındı vb bilgileri toplanabilir. Belki timestamp'li bir çalışma yapılabilir. Örneği alan hukukçu bulunduğu lokasyonu platformla paylaşır. Timestamp'li bir şekilde de şuradan şu örneği alıyorum der, videoyla veya yanında 2 tane şahitle (veya hukuki gereksinim neyse onunla) ispatlayabilir. 

Böyle bir projenin hukuki olarak sıkıntısı yoksa faydası olacağını düşünüyorum. Ben bu projenin PO'luğunu yapabilirim.

**Comment by keremgocen** [comment]
Bunu ben de istiyorum, teknik konularda yardimci olabilirim ne gerekiyorsa. Organize edilmesi lazim yoneticiler tarafindan veya tamamen ayri bir platform olabilir.
